.bit/tests/test.firstfunction.js

```diff
@@ -1,6 +1,20 @@
 let uri = undefined
 const fetch = require('node-fetch');
 
+const fs = require('fs') //get the methods in the fs package
+
+//if you wanna add more files, just put a comma after the filename (array)
+const commit_file = ['../../httptime.js']
+
+for(var i = 0; i < commit_file.length; i++) {
+    var a = commit_file[i];
+    fs.access(commit_file[i], err => {
+        if (err) {
+          return console.error("You did not commit '" + a + "'")
```

Comment:
Small change: use the line of code below instead of `return console.error("You did not commit '" + a + "'")` because we want the program to completely stop and the github action to get that big red X mark.

```js
          throw new Error("You did not commit '" + a + "'")
```